plaidml/bridge/openvino/ops/prior_box.cpp

```diff
@@ -0,0 +1,447 @@
+      float box_height = box_width;
+      float first = -box_width / conf.IW;
+      float second = -box_height / conf.IH;
+      box_max.push_back(first);
+      box_max.push_back(second);
+      box_max.push_back(-first);
+      box_max.push_back(-second);
+    }
+    TensorShape shape_max(DType::FLOAT32, {result_size, 4});
+    Buffer buffer_max(shape_max);
+    buffer_max.copy_from(box_max.data());
+    auto B_max = edsl::Constant(buffer_max, "box_max");
+
+    C_max_size = op::repeat(op::unsqueeze(C_mix_box, {-2})).count(result_size).axis(-2) + B_max;
+
+    // In this case, process index which min_size and max_size both have
+    if (conf.min_element_size <= max_element_size) {
+      C_out = op::concatenate({C_min_size, C_max_size}, -1);
+    } else {
+      C_out = op::concatenate(
+          {edsl::reshape(
+               op::concatenate(
+                   {op::slice(C_min_size).add_dim(0, conf.H).add_dim(0, conf.W).add_dim(0, result_size).add_dim(0, 4),
+                    C_max_size},
+                   -1),
+               {conf.H, conf.W, result_size * 8}),
+           edsl::reshape(op::slice(C_min_size)
+                             .add_dim(0, conf.H)
+                             .add_dim(0, conf.W)
+                             .add_dim(result_size, conf.min_element_size)
+                             .add_dim(0, 4),
+                         {conf.H, conf.W, (conf.min_element_size - result_size) * 4})},
+          -1);
+    }
+  }
+
+  // Aspect_ratio
+  if (conf.aspect_ratios_scale_size.size() > 0) {
+    std::vector<float> min_size_ar;
+    if (attrs.scale_all_sizes) {
+      for (auto ms : conf.min_size) min_size_ar.push_back(ms);
+    } else {
+      min_size_ar.push_back(conf.min_size[0]);
+    }
+    std::vector<float> box_ar;
+    for (auto ms : min_size_ar) {
+      for (auto ar : conf.aspect_ratios_scale_size) {
+        float box_width = ms * 0.5f * ar;
+        float box_height = ms * 0.5f / ar;
+        float first = -box_width / conf.IW;
+        float second = -box_height / conf.IH;
+        box_ar.push_back(first);
+        box_ar.push_back(second);
+        box_ar.push_back(-first);
+        box_ar.push_back(-second);
+      }
+    }
+    int msa_size = min_size_ar.size();
+    int ar_size = conf.aspect_ratios_scale_size.size();
+    TensorShape shape(DType::FLOAT32, {msa_size * ar_size, 4});
+    Buffer buffer_ar(shape);
+    buffer_ar.copy_from(box_ar.data());
+    auto B_ar = edsl::Constant(buffer_ar, "box_ar");
+
+    auto C_ar = op::repeat(op::unsqueeze(C_mix_box, {-2})).count(msa_size * ar_size).axis(-2) + B_ar;
+    if (attrs.scale_all_sizes) {
+      // In order to append results of aspect_ratio after former step, do reshape here
+      auto C_ar_reshape = edsl::reshape(C_ar, {conf.H, conf.W, conf.min_element_size, 4 * ar_size});
+      if (conf.min_element_size <= max_element_size) {
+        // When one min_size element work with one max_size element, just combine
```

Comment:
This comment needs some rewording. But I think the code is quite self-explainary, we can just delete it.